src/api/itemGeolocation.ts

```diff
@@ -78,3 +80,19 @@ export const getAddressFromCoordinates = async (
       },
     )
     .then(({ data }) => data);
+
+export const getSuggestionsForAddress = async (
```

Comment:
Are these suggestions or coordinates ?

Reply:
Suggestions of addresses, there's address label, and also lat lng.